scripts/TRN(registry data).R

```diff
@@ -0,0 +1,532 @@
+## Script to combine all TRNs of IntoValue and EU trials.
+## The TRN (registry data) table,  will be a list of these TRNs along with the other TRNs that are mentioned in their
+## registries, and also the sponsor protocol number associated with each IV or EU TRN, if available. We will also record which TRNs are in IV
+## or not, so we can easily prioritize potential cross-registrations later.
+
+##########################################################
+
+library(dplyr)
+library(tidyr)
+library(readr)
+library(here)
+library(fs)
+library(lubridate)
+library(stringr)
+library(ctregistries)
+library(cli)
+
+dir_raw <- here("data", "raw")
+dir_processed <- here("data", "processed")
+
+# full set of IV trials downloaded here so we can easily check which TRNs are in IV
+trials <- read_csv(path(dir_processed, "trials.csv"))
+cross_registrations <- read_rds(path(dir_processed, "trn", "cross-registrations.rds"))
+
+## download EU trial protocol data dump
+# Save in 'intovalue-data/data/raw/registries/euctr' to work here
+EU_protocol_dump <- read_csv(path(dir_raw, "registries/euctr/euctr_euctr_dump-2024-02-03-054239.csv"))
+
+## download EU trial results data dump
+# Save in 'intovalue-data/data/raw/registries/euctr' to work here
+# There is at least one trial that is in EU_results that is NOT in EU_protocol: 2006-005253-30
+# In lieu of a better solution, I'll just delete this row from EU_results for now.
+
+
+EU_results_dump <- read_csv(path(dir_raw, "registries/euctr/euctr_data_quality_results_scrape_feb_2024.csv" ))
```

Comment:
@lucfernandez remove trailing space after quote. you can check out the [lintr package](https://lintr.r-lib.org/) for more style feedback.